molgenis-core-ui/src/main/resources/templates/molgenis-header.ftl

```diff
@@ -207,6 +207,7 @@
 
                 <#if authenticated?? && authenticated>
                     <form id="logout-form" class="navbar-form navbar-right" method="post" action="/logout">
+                        <a id="manual" href="https://www.gitbook.com/book/molgenis/molgenis/details" target="_blank" class="btn btn-secondary">Manual</a>
```

Comment:
- Link should be https://molgenis.gitbooks.io/molgenis/content/
- Maybe 'documentation' is nicer as a link text?